internal/controller/postgrescluster/pgmonitor.go

```diff
@@ -342,14 +340,23 @@ func addPGMonitorExporterToInstancePodSpec(
 	}
 	template.Spec.Volumes = append(template.Spec.Volumes, configVolume)
 
-	// add default exporter queries config volume
-	queriesVolume := corev1.Volume{Name: "exporter-queries"}
-	queriesVolume.ConfigMap = &corev1.ConfigMapVolumeSource{
-		LocalObjectReference: corev1.LocalObjectReference{
-			Name: exporterQueriesConfig.Name,
-		},
+	// The original "custom queries" ability allowed users to provide a file with custom queries;
+	// however, it would turn off the default queries. The new "custom queries" ability allows
+	// users to append custom queries to the default queries. This new behavior is feature gated.
+	// Therefore, we only want to add the default queries ConfigMap as a source for the
+	// "exporter-config" volume if the AppendCustomQueries feature gate is turned on OR if the
+	// user has not provided any custom configuration.
+	if util.DefaultMutableFeatureGate.Enabled(util.AppendCustomQueries) ||
+		cluster.Spec.Monitoring.PGMonitor.Exporter.Configuration == nil {
+
+		configVolume.VolumeSource.Projected.Sources = append(configVolume.VolumeSource.Projected.Sources, corev1.VolumeProjection{
+			ConfigMap: &corev1.ConfigMapProjection{
+				LocalObjectReference: corev1.LocalObjectReference{
+					Name: exporterQueriesConfig.Name,
+				},
+			},
+		})
```

Comment:
:pick: in other code around this we have been using this pattern - i'd prefer if we stuck to that to be consistent
```suggestion
		volume := corev1.VolumeProjection{
			ConfigMap: &corev1.ConfigMapProjection{
				LocalObjectReference: corev1.LocalObjectReference{
					Name: exporterQueriesConfig.Name,
				},
			},
		}
        configVolume.VolumeSource.Projected.Sources = append(configVolume.VolumeSource.Projected.Sources, volume)
```